.agent-skill-chain/adapters/claude.sh

```
@@ -755,6 +760,94 @@ _stop_dispatch_lease_renew_daemon() {
+  _ASC_SESSION_DETACH_MODE=""
+
+  if command -v setsid >/dev/null 2>&1; then
+    _ASC_SESSION_DETACH_CMD=(setsid)
+    _ASC_SESSION_DETACH_MODE=setsid
+    return 0
+  fi
+
+  if command -v perl >/dev/null 2>&1 && perl -MPOSIX -e 'exit(defined(&POSIX::setsid) ? 0 : 1)' >/dev/null 2>&1; then
+    _ASC_SESSION_DETACH_CMD=(
+      perl -MPOSIX -e
+      'my $sid = POSIX::setsid(); die "setsid: $!\n" if !defined($sid) || $sid < 0; exec { $ARGV[0] } @ARGV; die "exec: $!\n";'
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)